components/Btn/Btn.css

```diff
@@ -163,11 +156,22 @@
   color: color(var(--color-error) a(70%));
 }
 
+/* Modifiers */
+
 .fullWidth {
   width: 100%;
   text-align: center;
 }
 
+.cta {
+  font-family: var(--font-brandon);
+  font-size: 0.9em;
```

Comment:
Personally think 0.8em would make them too small against the rest. @scott-riley for final say I guess?

Code good otherwise?